### PR TITLE
Ft map filter enhancement

### PR DESCRIPTION
**Pull request by FaithDaka**
#### Summary of Changes (What does this PR do?)
This PR includes a change to the map filter icon and addition of UX elements to make it visible.

#### Is this change ready to hit production in its current state?
- [x] Yes
- [ ] No

#### Status of maturity (all need to be checked before merging):

- [ ] I consider this code done
- [x] I've tested this locally
- [ ] This branch has been LGTM'ed by a reviewer

#### How should this be manually tested?
This branch should be run and changes will be see on the device management module

#### What are the relevant tickets?
- Github Issue: #505

#### Screenshots 
Screenshot (37)

### CODE REVIEW

**Comment by Codebmk** [comment]
I don't know if it's just me but the `support and feedback button` is covering the filter icon.

**Review by ja-odur — APPROVED** [review]
LGTM

**Comment by Baalmart** [comment]
> I don't know if it's just me but the `support and feedback button` is covering the filter icon.

We prolly need to find a more appropriate location for it? Midway of the screen? cc: @FaithDaka

File: netmanager/src/views/components/DataDisplay/Map/MapFilter.js [hunk 1]
```diff
@@ -18,19 +18,28 @@ import "assets/css/map-filter.css";
...
       <path
-        d="M10 18h4v-2h-4v2zM3 6v2h18V6H3zm3 7h12v-2H6v2z"
-        stroke={stroke || "white"}
-        fill={fill || "white"}
-      />
+      stroke={stroke || "white"}
+      fill={fill || "white"}/>
     </svg>
+    // <svg
+    //   xmlns="http://www.w3.org/2000/svg"
```
Comment (Baalmart) [inline netmanager/src/views/components/DataDisplay/Map/MapFilter.js:31]: Thanks! Could we avoid commented out code? Unless it is temporarily disabling some functionality? @FaithDaka

**Review by Baalmart — CHANGES_REQUESTED** [review]
Good work, thanks @FaithDaka !
From a UX perspective, I still feel that the feature (NOT the icon) is still not easily recognisable. Still requires some "repositioning" of sorts to fully address original concern.

**Please review attached screenshot** which shows my current suggestion---open to any other suggestion which addresses the hidden nature of this filter.
Screenshot 2022-03-08 at 10 40 46

**Comment by Baalmart** [comment]
Quick general comment, please add a `label` to the PR. Is it a WIP or ready for review? @FaithDaka Thanks again!

File: netmanager/src/views/components/DataDisplay/Map/MapFilter.js (hunk 1 above)
Comment (FaithDaka) [reply netmanager/src/views/components/DataDisplay/Map/MapFilter.js:31]: It was the previous icon's code. Just incase this was rejected

**Comment by FaithDaka** [comment]
> 

Sadly, the position for the icon and key is very static for an unknown reason (I checked both the file and in-line css). It would require brute force to move its position (I'd also wanted that) .

**Comment by FaithDaka** [comment]
Reviewed Implementation:

Screenshot (45)

**Review by Baalmart — APPROVED** [review]
LGTM

**Pull request merged**